src/drivers/bitbucket_cloud.js

```diff
@@ -21,13 +21,35 @@ class BitBucketCloud {
     const { project_path } = this;
     const { commit_sha, report } = opts;
 
-    // Print some variables
-    const endpoint = `/repositories/${project_path}/commit/${commit_sha}/comments/`;
-    const body = JSON.stringify({ content: { raw: report } });
-
-    const output = await this.request({ endpoint, method: 'POST', body });
-
-    return output;
+    // Make a comment in the commit
+    const commit_endpoint = `/repositories/${project_path}/commit/${commit_sha}/comments/`;
+    const commit_body = JSON.stringify({ content: { raw: report } });
+    const commit_output = await this.request({
+      endpoint: commit_endpoint,
+      method: 'POST',
+      body: commit_body
+    });
+
+    // Check for a corresponding PR. If it exists, also put the comment there.
+    const get_pr_endpt = `/repositories/${project_path}/commit/${commit_sha}/pullrequests`;
+    const pr_out = await this.request({ endpoint: get_pr_endpt });
+    if (pr_out.values) {
```

Comment:
maybe better 

```js
if pr_out.values && pr_out.values.length
```

this way we can control that pr_out is having values and values is bigger than zero. Not sure that the api is returning empty arrays?